bin/fin

```diff
@@ -4747,7 +4803,7 @@ run_cli ()
 	done
 
 	# Set default image
-	local RUN_IMAGE="${image:-docksal/cli:2.5-php7.1}"
+	local RUN_IMAGE="${image:-docksal/cli:2.6-php7.2}"
```

Comment:
@lmakarov so I propose we replace this with the new `cli:2-php7.2` tag. Cases like this were the main reason for its creation. Agreed or any comments?

Reply:
I'm fine with this and agree with it. We are running into issues where a different version is constantly being used all the time.